src/ALB/README.md

```diff
@@ -0,0 +1,164 @@
+<!-- region Generated -->
+# Az.Alb
+This directory contains the PowerShell module for the Alb service.
+
+---
+## Status
+[![Az.Alb](https://img.shields.io/powershellgallery/v/Az.Alb.svg?style=flat-square&label=Az.Alb "Az.Alb")](https://www.powershellgallery.com/packages/Az.Alb/)
```

Comment:
Please change all your ALB to Alb including the module folder. Follow this branch: https://github.com/Azure/azure-powershell/pull/22359/files